docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -2856,12 +2856,13 @@ If, for some reason the portal is modified with content that needs to be accesse
 Firewall SSO Integration
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-PacketFence is able to update some Firewalls based on device informations, like the IP address, the username connected on it. (Look for Integration guides to see how you can configure your Firewall with PacketFence)
-By default PacketFence use the dhcp traffic to trigger an update on the firewall but it's also possible to do it with the RADIUS Accounting traffic.
+PacketFence is able to update some firewall based on device informations, like the IP address, the username connected on it. (Look for Integration guides to see how you can configure your Firewall with PacketFence)
```

Comment:
Sorry my previous comments weren't clear.

its "some firewalls" and then later its "how you can configure your firewall with PacketFence"